meteor/lib/api/mediaManager.ts

```diff
@@ -1,22 +1,8 @@
+import * as Shared from '@sofie-automation/shared-lib/dist/peripheralDevice/mediaManager'
 export namespace MediaManagerAPI {
```

Comment:
Isn't this class redundant now and the places that need the WorkStep Enums should just import them from where they reside now?